documentation/src/mkdocs.yml

```diff
@@ -32,6 +32,7 @@ pages:
       - 'Working with Reader Group notifications': 'reader-group-notifications.md'
       - 'Working with State Synchronizer': 'state-synchronizer.md'
       - 'Working with Transactions': 'transactions.md'
+      - 'Working with StreamCuts': 'streamCuts.md'
```

Comment:
The file name is in camel case (streamCuts.md). However, in the rest of documents the format followed is that file names are lowercase and words separated by hyphens.

Reply:
renamed it to `streamcuts.md`